src/content/1.6/code/agda/snippet04.agda

```diff
@@ -1,2 +1,2 @@
-alpha : ((a × b) × c) → (a × (b × c))
-alpha ((x , y) , z) = x , (y , z)
+alpha : (A × B) × C → A × B × C
```

Comment:
I think we should drop extraneous parentheses whenever possible, unless they are present to illustrate a point.